crates/sui-framework/packages/sui-framework/sources/sui.move

```diff
@@ -54,4 +54,9 @@ module sui::sui {
     public entry fun transfer(c: coin::Coin<SUI>, recipient: address) {
         transfer::public_transfer(c, recipient)
     }
+
+    #[test_only]
+    public fun new_for_testing(ctx: &mut TxContext): Balance<SUI> {
```

Comment:
Could you add a doc-string here explaining the motivation for this function vs using the coin and balance test functions?

Reply:
I added a doc-string explaining the reason behind this update. Please let me know if it is clear.

Reply:
Looks great, thanks!